.docker/Dockerfile-template.ckan

```diff
@@ -1,18 +1,39 @@
-FROM openknowledge/ckan-dev:{CKAN_VERSION}
+FROM ckan/ckan-dev:{CKAN_VERSION}
+
+# swap between root and unprivileged user
+ARG ORIGINAL_USER
+RUN ORIGINAL_USER=$(id -un)
 
 ARG SITE_URL=http://ckan:5000/
 ENV PYTHON_VERSION={PYTHON_VERSION}
 ENV CKAN_VERSION={CKAN_VERSION}
+ENV SOLR_VERSION={SOLR_VERSION}
 ENV CKAN_SITE_URL="${SITE_URL}"
 ENV PYTHON={PYTHON}
 
 WORKDIR "${APP_DIR}"
 
-ENV DOCKERIZE_VERSION v0.6.1
-RUN apk add --no-cache build-base \
-    && curl -sL https://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz \
+COPY .docker/ckan.ini $CKAN_INI
+
+COPY . "${APP_DIR}"/
+
+USER root
+
+COPY bin/ckan_cli /usr/bin/
+
+RUN chmod +x "${APP_DIR}"/bin/*.sh /usr/bin/ckan_cli
```

Comment:
can you pull the updated way from ckanext-validation plz.

Reply:
Are you referring to Dockerize? (If so, this comment was on the wrong line.)

I'm reluctant to include 30 lines of shell code just to increment the version of something that we only use once to check if a container has started.

Reply:
it also handles ubuntu vs alpine installs for any other items we want like curl etc. It handles arm64 and amd64 images (apple vs rest)